packages/schema/src/schema.ts

```diff
@@ -97,4 +97,5 @@ export type MaruSongGist = InferInput<typeof MaruSongGistSchema>
 
 export type MaruSongDataParsed = MaruSongData & {
   lyrics: LyricLine[]
+  lyricLocales: string[]
```

Comment:
I don't think we need this extra data. It should be easily got with a single computed.